app/javascript/src/constants/formInitialValues.js

```diff
@@ -16,5 +16,7 @@ export default {
   notesForm: {
     title: "",
     description: "",
+    contact: "",
+    tags: "",
   },
 };
```

Comment:
Please look into my suggestion for moving this to a constants file in the scope of this directory.